Gemfile

```diff
@@ -3,6 +3,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
```

Comment:
- [ ] It would be nice if you make a new pull request because you are repeating the block so it will break the consistency for your work and for review consistency too 😨 So it would be better if you make a new pull request and submit it for a review 👍